File: MODUL 3/SOAL 5/PRAK205-2210817210026-Zulfa Auliya Akbar.py
import math

A,B = map(float,input().split())

print("Alas =",round(math.sqrt((B*B)-(A*A))),"cm" )
print("tinggi =",round(A),"cm")
print("Keliling =",round(A+B+math.sqrt((B*B)-(A*A))),"cm")
print("Luas =",round(0.5*A*(math.sqrt((B*B)-(A*A)))),"cm^2")

File: MODUL 3/SOAL 5/PRAK205-2210817210026-Zulfa Auliya Akbar.c
#include <stdio.h>
#include <math.h>
int main(){

    float A, B;
    scanf("%f \n %f",&A, &B);
    printf("Alas = %.f cm \n", sqrt((B*B)-(A*A)) );
    printf("tinggi = %.f cm \n", A);
    printf("Keliling = %.f cm \n", A+B+sqrt((B*B)-(A*A)));
    printf("Luas = %.f cm^2", 0.5*A*(sqrt((B*B)-(A*A))));

    return 0;
}
